model/src/invite/stage_instance.rs

```diff
@@ -0,0 +1,84 @@
+use crate::{id::RoleId, user::User};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct InviteStageInstance {
+    /// Members speaking in the Stage.
+    pub members: Vec<InviteStageInstanceMember>,
+    /// Total number of users.
+    pub participant_count: u64,
+    /// Number of speakers.
+    pub speaker_count: u64,
+    /// Topic of the Stage instance.
+    ///
+    /// Between 1 and 120 characters.
+    pub topic: String,
+}
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct InviteStageInstanceMember {
+    /// Avatar hash.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub avatar: Option<String>,
+    /// ISO8601 timestamp of the date the member joined the guild.
```

Comment:
ISO 8601 stuff like @zeylahellyer noted in the cache PR

Reply:
thanks